install/netbird-install.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: TechHutTV
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://netbird.io/
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+app="netbird"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  ca-certificates \
+  gnupg
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up NetBird Repository"
+curl -sSL https://pkgs.netbird.io/debian/public.key | gpg --dearmor -o /usr/share/keyrings/netbird-archive-keyring.gpg
+chmod 0644 /usr/share/keyrings/netbird-archive-keyring.gpg
+echo 'deb [signed-by=/usr/share/keyrings/netbird-archive-keyring.gpg] https://pkgs.netbird.io/debian stable main' | tee /etc/apt/sources.list.d/netbird.list >/dev/null
+$STD apt-get update
+msg_ok "Set up NetBird Repository"
+
+msg_info "Installing NetBird"
+$STD apt-get install -y netbird
+msg_ok "Installed NetBird"
+
+msg_info "Enabling NetBird Service"
+$STD systemctl enable --now netbird
+msg_ok "Enabled NetBird Service"
+
+# NetBird Deployment Type Selection
+echo ""
+echo -e "${BL}NetBird Deployment Type${CL}"
+echo "─────────────────────────────────────────"
+echo "Are you using NetBird Managed or Self-Hosted?"
+echo ""
+echo " 1) NetBird Managed (default) - Use NetBird's managed service"
+echo " 2) Self-Hosted - Use your own NetBird management server"
+echo ""
+
+read -r -p "Select deployment type [1]: " DEPLOYMENT_TYPE
```

Comment:
```suggestion
read -r -p "${TAB3}Select deployment type [1]: " DEPLOYMENT_TYPE
```
$TAB3 aligns the text with rest of the status messages